### DISCUSSION TITLE
Integrity check failing for Picomatch

### DISCUSSION THREAD

**Discussion by brydonm** [biomejs/biome#3231] [Q&A] [answered]
https://github.com/micromatch/picomatch/issues/131

>Hey there,
>
>I'm trying to install @biomejs/biome via yarn add --dev --exact @biomejs/biome and I'm seeing this error:
>
>error https://registry.yarnpkg.com/picomatch/-/picomatch-2.3.1.tgz: Integrity check failed for "picomatch" (computed integrity doesn't match our records, got "sha512-JU3teHTNjmE2VCGFzuY8EXzCDVwEqB2a8fsIvwaStHhAWJEeVd1o1QD80CU6+ZdEXXSLbSsuLwJjkCBWqRQUVA== sha1-O6ODNzNkbZ0+SZWUbBNlpn+wekI=")
>
>Is this an issue with https://github.com/biomejs, or an issue in this repo, or locally?

Is anyone else running into the issue above?

**Comment by Sec-ant** [comment]
Does any of these links solve your problem?

https://www.google.com/search?q=yarn+Integrity+check+failed+for+site%3Astackoverflow.com

**Reply by brydonm** [reply]
Thanks, but no. I did google the issue and attempt most fixes that people suggested. Seems most of them are not trying to install a new package, but install dependencies that already exist.

**Reply by Sec-ant** [reply]
Can you provide your repo where you want to install this package? Is `picomatch` already one of the (direct or indirect) dependencies in that repo?

**Reply by brydonm** [reply, marked as answer]
It was included in the lockfile. I was hesitant to reset it but I did remove the lockfile and then it installed as normal. Thanks for your help!
